scripts/HO-example.jl

```diff
@@ -0,0 +1,112 @@
+# # Example 5: Harmonic oscillator
+
+# This is an example calculation of the harmonic oscillator with
+# contact interactions that are treated with first-order perturbation
+# theory.
+
+# A runnable script for this example is located
+# [here](https://github.com/joachimbrand/Rimu.jl/blob/develop/scripts/HO-example.jl).
+# Run it with `julia HO-example.jl`.
+
+# Firstly, we load all needed modules.
+using Rimu
+using DataFrames
+using LinearAlgebra
+
+# First we define the system size for two particles in a 2D harmonic oscillator
+# allowing `M` levels above the ground state in each dimension.
+N = 2
+D = 2
+M = 3
+S = ntuple(_ -> M + 1, D)
+P = prod(S)
```

Comment:
Can you explain what `N` and `D` stand for?

It is not so easy to parse what `S` will be, so maybe consider printing it, or defining it at the end of a block, such that the value appears in the html output.

Another option would be to define it by giving the explicit values `S = (M+1, M+1)`. Then you would not have to introduce the variable `D` and everything will be easier to read.

Reply:
I have simplified this setup section